lib/serverpool/serverpool.c

```diff
@@ -269,6 +269,9 @@ forkdns(const char * target, int readfd, int writefd, int freq)
 	size_t addrlen;
 	size_t i;
 
+	/* Sanity check. */
+	assert(freq >= 0);
```

Comment:
Is there any reason to not simply make `freq` unsigned?

Reply:
That was my preferred solution, but I thought it would provoke too many changed lines.  I'm changing it now.